src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableClient.cs

```diff
@@ -358,28 +358,44 @@ async Task<DurableOrchestrationStatus> IDurableOrchestrationClient.GetStatusAsyn
         /// <inheritdoc />
         async Task<IList<DurableOrchestrationStatus>> IDurableOrchestrationClient.GetStatusAsync(CancellationToken cancellationToken)
         {
-            IList<OrchestrationState> states = await this.DurabilityProvider.GetAllOrchestrationStates(cancellationToken);
-
-            var results = new List<DurableOrchestrationStatus>();
-            foreach (OrchestrationState state in states)
-            {
-                results.Add(ConvertOrchestrationStateToStatus(state));
-            }
-
-            return results;
+            return await this.GetAllStatusHelper(null, null, null, cancellationToken);
         }
 
         /// <inheritdoc />
         async Task<IList<DurableOrchestrationStatus>> IDurableOrchestrationClient.GetStatusAsync(DateTime createdTimeFrom, DateTime? createdTimeTo, IEnumerable<OrchestrationRuntimeStatus> runtimeStatus, CancellationToken cancellationToken)
         {
-            IList<OrchestrationState> states = await this.DurabilityProvider.GetAllOrchestrationStatesWithFilters(createdTimeFrom, createdTimeTo, runtimeStatus, cancellationToken);
```

Comment:
Does your change also allow us to deprecate/remove `GetAllOrchestrationStatesWithFilters`?

Reply:
I didn't notice this before, but yes, it does.